### ISSUE TITLE
Dialogs not launching

### ISSUE THREAD

**Issue by docwilmot** [backdrop-contrib/gin#33]
I get error in console
```
Uncaught TypeError: The element must be an instance of Element
    at u (once.js?v=1.25.x-dev:2:382)
```
`themes/gin/js/libs/once.js`

**Comment by docwilmot** [comment]
Actually the modal opens below the page: (Views "Add field" dialog)

![fullscreenpage](https://github.com/backdrop-contrib/gin/assets/1042283/91786c65-457a-4c74-950e-c0f54e121e6b)

**Comment by docwilmot** [comment]
OK, I updated to the very latest dev of Backdrop and flushed caches and dialogs now launch fine. Keeping in mind my local was last fetched only 1 month ago, so it probably might be a problem? Leaving this open in case but feel free to close.

**Comment by saschaeggi** [comment]
@docwilmot modals have been fixed with df9e8af

Please note that we're in a very early unstable state where we have maybe ported 30% of things. So expect a lot of bugs until we're "done" with migrating things. This is a work in progress.

**Comment by docwilmot** [comment]
I recognized the early state of the project, thanks for all your work. Usually if there is not an open issue for a bug I find I'll open an issue, in case its not apparent to the devs, or its only on my configuration etc.
